lib/rbtrace/interactive/irb.rb

```diff
@@ -10,13 +10,13 @@ def print(arg=nil)
     RBTraceCLI.tracer.print(arg)
   end
 
-  def inspect_last_value
+  def inspect_last_value(output = +"")
     @last_value
   end
 end
 
 class IRB::WorkSpace
-  def evaluate(context, statements, file=__FILE__, line=__LINE__)
+  def evaluate(statements, file=__FILE__, line=__LINE__)
```

Comment:
Current signature of this method in IRB:

https://github.com/ruby/irb/blob/feeb7be6795271f51e5c3d75f45eacdb491a34fa/lib/irb/workspace.rb#L100